pkgs/tools/networking/redsocks/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, libevent }:
+
+stdenv.mkDerivation rec {
+  name = "redsocks-0.5";
+
+  src = fetchFromGitHub {
+    owner = "darkk";
+    repo = "redsocks";
+    rev = "release-0.5";
```

Comment:
Consider factoring out the version so that it can be shared between `name` and `rev`.